### ISSUE TITLE
Cannot set expiration for long period

### ISSUE THREAD

**Issue by steache** [dragonflydb/dragonfly#2245]
**Describe the bug**
 - I know the title of this issue sound silly, but official redis & many of drop in replacements allow this feature.

**To Reproduce**
- SET test "test"
- EXPIRE test 4855058247

**Expected behavior**
- work

**Screenshots**
![Screenshot from 2023-12-01 20-35-19](https://github.com/dragonflydb/dragonfly/assets/34166829/ca380e60-8760-4eb6-a573-bff053fb330d)

**Reproducible Code Snippet**
```
SET test "test"
EXPIRE test 4855058247
```

Thank you for any response

**Comment by romange** [comment]
yes, we do not plan to support this.   Why do you need expiry so long @steache ?

**Comment by steache** [comment]
@romange Thank you for your response. :)

To be honest, I think it is an anti pattern to set the expiration so high also.

Also I was thinking of even posting this as issue/bug, I get it, as it is not normal usage, but redis supports it.

We unfortunately have few legacy apps, that do this kind of wrong use (and we will fix them, so we can still use dragonflydb, because I think it is pretty good alternative to redis) but I was thinking of other uses, that might switch from redis to dragonfly and don't realize, that there are some caveats for the "drop in replacement" -> it has some limitations (even if you try to specify each & every command and it's level of implementation via "supported/partially supported etc". :)

Don't get me wrong, I really like this project.

I absolutely get your answer and respect your decision not to support it.

But maybe if I could ask you to specify some mild variantions/diffs from redis, so another person does not fall into this hole. (as we did and as a result of this, we were charged ~$20k on gcp for not using cache)

**Comment by romange** [comment]
Yeah, sounds familiar 🥲  I also wasted lots of dollars on cloud providers for things that could be easily avoided. My first lesson on AWS was that we used public ip addresses between backends and then we got the bill for egress data charges 🤦🏼‍♂️ 

Regarding the differences - we  have this document https://github.com/dragonflydb/dragonfly/blob/main/docs/differences.md
but it's probably makes sense to document it in command reference docs as well.

**Comment by steache** [comment]
@romange Thank you, I did not stumble upon this document.

So this is all solved, thank you very much and again thank you for this project! :)
